pyrit/prompt_converter/llm_generic_text_converter.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+import uuid
+
+from pyrit.models import PromptDataType, PromptRequestPiece, PromptRequestResponse, PromptTemplate
+from pyrit.prompt_converter import PromptConverter, ConverterResult
+from pyrit.prompt_target import PromptChatTarget
+
+logger = logging.getLogger(__name__)
+
+
+class LLMGenericTextConverter(PromptConverter):
```

Comment:
```suggestion
class GenericTextConverter(PromptConverter):
```

Nit: how about that?

Reply:
Oof, it does sound cleaner, but I think I prefer `LLMGenericTextConverter` because `GenericTextConverter` sounds more vague, and without context it feels difficult to know what it's doing

Reply:
I don't feel like the LLM part adds clarity but it is a bit and you may ignore it 😄